core/templates/dev/head/pages/exploration_player/exploration_footer_directive.html

```diff
@@ -129,10 +130,11 @@ <h4 translate="I18N_PLAYER_SHARE_THIS_EXPLORATION" class="share-exploration-foot
     line-height: 0.6;
     padding-left: 4px;
     text-overflow: clip;
+    text-align: center;
```

Comment:
Thanks @markhalpin! Here is how  footer will look without center align on hover : 
![longwidth](https://cloud.githubusercontent.com/assets/14854536/21740446/86e982ba-d4e0-11e6-97a2-d47e169284b1.png)